keyboards/xd68/keymaps/default/keymap.c

```diff
@@ -0,0 +1,48 @@
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /* Keymap _BL: (Base Layer) Default Layer
+   * ,----------------------------------------------------------------.
+   * |Esc | 1|  2|  3|  4|  5|  6|  7|  8|  9|  0|  -|  =|Backsp |Home|
+   * |----------------------------------------------------------------|
+   * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|  \  |PgUp|
+   * |----------------------------------------------------------------|
+   * |CAPS   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Return |PgDn|
+   * |----------------------------------------------------------------|
+   * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift | Up|End |
+   * |----------------------------------------------------------------|
+   * |Ctrl|Win |Alt |        Space          |App|Ctrl|FN |Lef|Dow|Rig |
+   * `----------------------------------------------------------------'
+   */
+[_BL] = LAYOUT_65_ansi(
+  KC_ESC,    KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_BSPC,  KC_HOME, 
+  KC_TAB,    KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,KC_BSLS,  KC_PGUP, 
+  KC_CAPS,   KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,KC_QUOT,         KC_ENT,   KC_PGDN,  
+  KC_LSFT,   KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT, KC_SLSH,KC_RSFT,           KC_UP,  KC_END, 
+  KC_LCTL,   KC_LGUI,KC_LALT,                KC_SPC,                         KC_APP, KC_RCTRL, MO(_FL), KC_LEFT,KC_DOWN,KC_RGHT
+  ),
+
+  /* Keymap _FL: Function Layer
+   * ,----------------------------------------------------------------.
+   * |~ `| F1|F2 |F3 |F4 |F5 |F6 |F7 |F8 |F9 |F10|F11|F12|Del    |Ins |
+   * |----------------------------------------------------------------|
+   * |     |   |Up |   |   |   |   |   |   |   |   |   |   |     |    |
+   * |----------------------------------------------------------------|
+   * |      |<- |Dn | ->|   |   |   |   |   |   |   |   |        |    |
+   * |----------------------------------------------------------------|
+   * |HUI   |SAI|VAI|RGBMOD|BL-|BL |BL+|   |   |   |   |Play |V+ |Mut |
+   * |----------------------------------------------------------------|
+   * |HUD |SAD |VAD |       RGB_Tog         |   |   |    | <M|V- |M>  |
+   * `----------------------------------------------------------------'
+   */
+[_FL] = LAYOUT_65_ansi(
+  KC_GRV, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_DEL, KC_INS,  
+  _______,_______, KC_UP,_______,_______, _______,_______,_______,_______,_______,_______,_______,_______, _______,_______, 
```

Comment:
add a RESET code to the default keymap, this makes getting into bootloader easier. Recommend |\ or backspace.

Reply:
Hi, 

Added those changes, thanks for the review.